app/controllers/admin/api/backend_apis_controller.rb

```diff
@@ -8,7 +8,7 @@ class Admin::Api::BackendApisController < Admin::Api::BaseController
   clear_respond_to
   respond_to :json
 
-  wrap_parameters BackendApi
+  wrap_parameters BackendApi, include: BackendApi.attribute_names | %w[annotations]
```

Comment:
Do we need this? In `Annotating::Model` we have:
```
      has_many :annotations, as: :annotated, dependent: :destroy, autosave: true
```

Doesn't that make `annotations` already part of `#attribute_names`?